react/tic-tac-toe/src/app/screens/Game/components/Square/index.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
-
 import styles from './styles.module.scss';
 
-const Square = () => (
-  <button type="button" className={styles.square}>
-    {/* TODO */}
-  </button>
-);
+function Square(props) {
```

Comment:
Here you can recieve props on more specific way like `function Square ({onClick})` and when you use it you avoid to write `props.onClick`